kubermatic-addons/addon-manifests/flux2.yaml

```diff
@@ -0,0 +1,231 @@
+apiVersion: kubermatic.k8s.io/v1
+kind: AddonConfig
+metadata:
+  name: flux2
+spec:
+  description: Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+  formSpec:
+      - displayName: Git repository URL
+        helpText: URL of the Git repository used as Flux source (only HTTP/S URLs are currently supported)
+        internalName: gitRepositoryUrl
+        required: true
+        type: text
+      - displayName: Git repository branch
+        helpText: Name of the Git branch to checkout and monitor for changes (defaults to "master")
+        internalName: gitRepositoryBranch
+        required: false
+        type: text
+      - displayName: Manifest directory
+        helpText: Relative path to the directory within the Git repository containing manifests to be reconciled in the cluster (defaults to "./clusters/${CLUSTER_ID}")
+        internalName: kustomizationPath
```

Comment:
kustomizationPath may be a bit misleading for users ... as the kustomize vs kustomization (flux one), let's keep using the `manifestsPath`

Reply:
I will also adjust the variable name for the ArgoCD addon to keep things consistent